packages/webdriverio/package.json

```diff
@@ -61,6 +61,7 @@
     "lodash.zip": "^4.2.0",
     "rgb2hex": "^0.1.0",
     "serialize-error": "^3.0.0",
+    "source-map-support": "^0.5.6",
```

Comment:
Why version __^0.5.6__, when `devDependencies` has version __^0.5.9__?
https://github.com/webdriverio/webdriverio/blob/c31a3a6583120ede3892b5248488eb34641c2395/package.json#L52

Or are you simply moving the dependency from wdio-cli to webdriver(io)?
https://github.com/webdriverio/webdriverio/blob/c31a3a6583120ede3892b5248488eb34641c2395/packages/wdio-cli/package.json#L52

Reply:
I simply tried to synchronise the dependency between the CLI and the standalone packages. Someone more qualified than I would have to advise about which version is preferable.